<ui version="4.0" >
 <class>TwodSelBase</class>
 <widget class="QDialog" name="TwodSelBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>2D Scan Combination</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QVBoxLayout" >
     <item>
      <layout class="QHBoxLayout" >
       <property name="spacing" >
        <number>9</number>
       </property>
       <property name="margin" >
        <number>2</number>
       </property>
       <item>
        <widget class="QLabel" name="textLabel1_2" >
         <property name="text" >
          <string>Label for scan item</string>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Q3TextEdit" name="scanLabel" >
         <property name="maximumSize" >
          <size>
           <width>32767</width>
           <height>30</height>
          </size>
         </property>
         <property name="text" >
          <string>new 2D scan</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Q3GroupBox" name="groupBox1" >
       <property name="title" >
        <string>Scan parameters</string>
       </property>
       <layout class="QVBoxLayout" >
        <item>
         <layout class="QVBoxLayout" >
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QLabel" name="textLabel1" >
              <property name="text" >
               <string>Scan variable</string>
              </property>
              <property name="wordWrap" >
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="scanSel" />
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QLabel" name="textLabel1_3_3" >
              <property name="text" >
               <string># steps</string>
              </property>
              <property name="wordWrap" >
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="spacer3_2_3" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType" >
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>81</width>
                <height>21</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QSpinBox" name="stepBox" >
              <property name="enabled" >
               <bool>false</bool>
              </property>
              <property name="palette" >
               <palette>
                <active>
                 <colorrole role="WindowText" >
                  <brush brushstyle="SolidPattern" >
                   <color alpha="255" >
                    <red>0</red>
                    <green>0</green>
                    <blue>0</blue>
                   </color>
                  </brush>
                 </colorrole>
                 <colorrole role="Button" >
                  <brush brushstyle="SolidPattern" >
                   <color alpha="255" >
                    <red>144</red>
                    <green>176</green>
                    <blue>168</blue>
                   </color>
                  </brush>
                 </colorrole>
                 <colorrole role="Light" >
                  <brush brushstyle="SolidPattern" >
                   <color alpha="255" >
                    <red>218</red>
                    <green>255</green>
                    <blue>246</blue>
                   </color>
                  </brush>
                 </colorrole>
                 <colorrole role="Midlight" >
                  <brush brushstyle="SolidPattern" >
                   <color alpha="255" >
                    <red>181</red>
                    <green>215</green>
                    <blue>207</blue>
                   </color>
                  </brush>
                 </colorrole>
                 <colorrole role="Dark" >
                  <brush brushstyle="SolidPattern" >
                   <color alpha="255" >
                    <red>72</red>
                    <green>88</green>
                    <blue>84</blue>
                   </color>
                  </brush>
                 </colorrole>
                 <colorrole role="Mid" >
                  <brush brushstyle="SolidPattern" >
                   <color alpha="255" >
                    <red>96</red>
                    <green>117</green>
                    <blue>112</blue>
                   </color>
                  </brush>
                 </colorrole>
                 <colorrole role="Text" >
                  <brush brushstyle="SolidPattern" >
                   <color alpha="255" >
                    <red>53</red>
                    <green>53</green>
                    <blue>53</blue>
                   </color>
                  </brush>
                 </colorrole>
                 <colorrole role="BrightText" >
                  <brush brushstyle="SolidPattern" >
                   <color alpha="255" >
                    <red>255</red>
                    <green>255</green>
                    <blue>255</blue>
                   </color>
                  </brush>
                 </colorrole>
                 <colorrole role="ButtonText" >
                  <brush brushstyle="SolidPattern" >
                   <color alpha="255" >
                    <red>0</red>
                    <green>0</green>
                    <blue>0</blue>
                   </color>
                  </brush>
                 </colorrole>
                 <colorrole role="Base" >
                  <brush brushstyle="SolidPattern" >
                   <color alpha="255" >
                    <red>255</red>
                    <green>255</green>
                    <blue>255</blue>
                   </color>
                  </brush>
                 </colorrole>
                 <colorrole role="Window" >
                  <brush brushstyle="SolidPattern" >
                   <color alpha="255" >
                    <red>214</red>
                    <green>214</green>
                    <blue>214</blue>
                   </color>
                  </brush>
                 </colorrole>
                 <colorrole role="Shadow" >
                  <brush brushstyle="SolidPattern" >
                   <color alpha="255" >
                    <red>0</red>
                    <green>0</green>
                    <blue>0</blue>
                   </color>
                  </brush>
                 </colorrole>
                 <colorrole role="Highlight" >
                  <brush brushstyle="SolidPattern" >
                   <color alpha="255" >
                    <red>0</red>
                    <green>0</green>
                    <blue>128</blue>
                   </color>
                  </brush>
                 </colorrole>
                 <colorrole role="HighlightedText" >
                  <brush brushstyle="SolidPattern" >
                   <color alpha="255" >
                    <red>255</red>
                    <green>255</green>
                    <blue>255</blue>
                   </color>
                  </brush>
                 </colorrole>
                 <colorrole role="Link" >
                  <brush brushstyle="SolidPattern" >
                   <color alpha="255" >
                    <red>0</red>
                    <green>0</green>
                    <blue>0</blue>
                   </color>
                  </brush>
                 </colorrole>
                 <colorrole role="LinkVisited" >
                  <brush brushstyle="SolidPattern" >
                   <color alpha="255" >
                    <red>0</red>
                    <green>0</green>
                    <blue>0</blue>
                   </color>
                  </brush>
                 </colorrole>
                </active>
                <inactive>
                 <colorrole role="WindowText" >
                  <brush brushstyle="SolidPattern" >
                   <color alpha="255" >
                    <red>0</red>
                    <green>0</green>
                    <blue>0</blue>
                   </color>
                  </brush>
                 </colorrole>
                 <colorrole role="Button" >
                  <brush brushstyle="SolidPattern" >
                   <color alpha="255" >
                    <red>144</red>
                    <green>176</green>
                    <blue>168</blue>
                   </color>
                  </brush>
                 </colorrole>
                 <colorrole role="Light" >
                  <brush brushstyle="SolidPattern" >
                   <color alpha="255" >
                    <red>218</red>
                    <green>255</green>
                    <blue>246</blue>
                   </color>
                  </brush>
                 </colorrole>
                 <colorrole role="Midlight" >
                  <brush brushstyle="SolidPattern" >
                   <color alpha="255" >
                    <red>166</red>
                    <green>202</green>
                    <blue>193</blue>
                   </color>
                  </brush>
                 </colorrole>
                 <colorrole role="Dark" >
                  <brush brushstyle="SolidPattern" >
                   <color alpha="255" >
                    <red>72</red>
                    <green>88</green>
                    <blue>84</blue>
                   </color>
                  </brush>
                 </colorrole>
                 <colorrole role="Mid" >
                  <brush brushstyle="SolidPattern" >
                   <color alpha="255" >
                    <red>96</red>
                    <green>117</green>
                    <blue>112</blue>
                   </color>
                  </brush>
                 </colorrole>
                 <colorrole role="Text" >
                  <brush brushstyle="SolidPattern" >
                   <color alpha="255" >
                    <red>53</red>
                    <green>53</green>
                    <blue>53</blue>
                   </color>
                  </brush>
                 </colorrole>
                 <colorrole role="BrightText" >
                  <brush brushstyle="SolidPattern" >
                   <color alpha="255" >
                    <red>255</red>
                    <green>255</green>
                    <blue>255</blue>
                   </color>
                  </brush>
                 </colorrole>
                 <colorrole role="ButtonText" >
                  <brush brushstyle="SolidPattern" >
                   <color alpha="255" >
                    <red>0</red>
                    <green>0</green>
                    <blue>0</blue>
                   </color>
                  </brush>
                 </colorrole>
                 <colorrole role="Base" >
                  <brush brushstyle="SolidPattern" >
                   <color alpha="255" >
                    <red>255</red>
                    <green>255</green>
                    <blue>255</blue>
                   </color>
                  </brush>
                 </colorrole>
                 <colorrole role="Window" >
                  <brush brushstyle="SolidPattern" >
                   <color alpha="255" >
                    <red>214</red>
                    <green>214</green>
                    <blue>214</blue>
                   </color>
                  </brush>
                 </colorrole>
                 <colorrole role="Shadow" >
                  <brush brushstyle="SolidPattern" >
                   <color alpha="255" >
                    <red>0</red>
                    <green>0</green>
                    <blue>0</blue>
                   </color>
                  </brush>
                 </colorrole>
                 <colorrole role="Highlight" >
                  <brush brushstyle="SolidPattern" >
                   <color alpha="255" >
                    <red>0</red>
                    <green>0</green>
                    <blue>128</blue>
                   </color>
                  </brush>
                 </colorrole>
                 <colorrole role="HighlightedText" >
                  <brush brushstyle="SolidPattern" >
                   <color alpha="255" >
                    <red>255</red>
                    <green>255</green>
                    <blue>255</blue>
                   </color>
                  </brush>
                 </colorrole>
                 <colorrole role="Link" >
                  <brush brushstyle="SolidPattern" >
                   <color alpha="255" >
                    <red>0</red>
                    <green>0</green>
                    <blue>255</blue>
                   </color>
                  </brush>
                 </colorrole>
                 <colorrole role="LinkVisited" >
                  <brush brushstyle="SolidPattern" >
                   <color alpha="255" >
                    <red>255</red>
                    <green>0</green>
                    <blue>255</blue>
                   </color>
                  </brush>
                 </colorrole>
                </inactive>
                <disabled>
                 <colorrole role="WindowText" >
                  <brush brushstyle="SolidPattern" >
                   <color alpha="255" >
                    <red>128</red>
                    <green>128</green>
                    <blue>128</blue>
                   </color>
                  </brush>
                 </colorrole>
                 <colorrole role="Button" >
                  <brush brushstyle="SolidPattern" >
                   <color alpha="255" >
                    <red>144</red>
                    <green>176</green>
                    <blue>168</blue>
                   </color>
                  </brush>
                 </colorrole>
                 <colorrole role="Light" >
                  <brush brushstyle="SolidPattern" >
                   <color alpha="255" >
                    <red>218</red>
                    <green>255</green>
                    <blue>246</blue>
                   </color>
                  </brush>
                 </colorrole>
                 <colorrole role="Midlight" >
                  <brush brushstyle="SolidPattern" >
                   <color alpha="255" >
                    <red>166</red>
                    <green>202</green>
                    <blue>193</blue>
                   </color>
                  </brush>
                 </colorrole>
                 <colorrole role="Dark" >
                  <brush brushstyle="SolidPattern" >
                   <color alpha="255" >
                    <red>72</red>
                    <green>88</green>
                    <blue>84</blue>
                   </color>
                  </brush>
                 </colorrole>
                 <colorrole role="Mid" >
                  <brush brushstyle="SolidPattern" >
                   <color alpha="255" >
                    <red>96</red>
                    <green>117</green>
                    <blue>112</blue>
                   </color>
                  </brush>
                 </colorrole>
                 <colorrole role="Text" >
                  <brush brushstyle="SolidPattern" >
                   <color alpha="255" >
                    <red>53</red>
                    <green>53</green>
                    <blue>53</blue>
                   </color>
                  </brush>
                 </colorrole>
                 <colorrole role="BrightText" >
                  <brush brushstyle="SolidPattern" >
                   <color alpha="255" >
                    <red>255</red>
                    <green>255</green>
                    <blue>255</blue>
                   </color>
                  </brush>
                 </colorrole>
                 <colorrole role="ButtonText" >
                  <brush brushstyle="SolidPattern" >
                   <color alpha="255" >
                    <red>128</red>
                    <green>128</green>
                    <blue>128</blue>
                   </color>
                  </brush>
                 </colorrole>
                 <colorrole role="Base" >
                  <brush brushstyle="SolidPattern" >
                   <color alpha="255" >
                    <red>255</red>
                    <green>255</green>
                    <blue>255</blue>
                   </color>
                  </brush>
                 </colorrole>
                 <colorrole role="Window" >
                  <brush brushstyle="SolidPattern" >
                   <color alpha="255" >
                    <red>214</red>
                    <green>214</green>
                    <blue>214</blue>
                   </color>
                  </brush>
                 </colorrole>
                 <colorrole role="Shadow" >
                  <brush brushstyle="SolidPattern" >
                   <color alpha="255" >
                    <red>0</red>
                    <green>0</green>
                    <blue>0</blue>
                   </color>
                  </brush>
                 </colorrole>
                 <colorrole role="Highlight" >
                  <brush brushstyle="SolidPattern" >
                   <color alpha="255" >
                    <red>0</red>
                    <green>0</green>
                    <blue>128</blue>
                   </color>
                  </brush>
                 </colorrole>
                 <colorrole role="HighlightedText" >
                  <brush brushstyle="SolidPattern" >
                   <color alpha="255" >
                    <red>255</red>
                    <green>255</green>
                    <blue>255</blue>
                   </color>
                  </brush>
                 </colorrole>
                 <colorrole role="Link" >
                  <brush brushstyle="SolidPattern" >
                   <color alpha="255" >
                    <red>0</red>
                    <green>0</green>
                    <blue>255</blue>
                   </color>
                  </brush>
                 </colorrole>
                 <colorrole role="LinkVisited" >
                  <brush brushstyle="SolidPattern" >
                   <color alpha="255" >
                    <red>255</red>
                    <green>0</green>
                    <blue>255</blue>
                   </color>
                  </brush>
                 </colorrole>
                </disabled>
               </palette>
              </property>
              <property name="minimum" >
               <number>2</number>
              </property>
              <property name="maximum" >
               <number>2056</number>
              </property>
              <property name="value" >
               <number>2</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="reverseScan" >
            <property name="text" >
             <string>scan points are in decreasing order</string>
            </property>
            <property name="checked" >
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QLabel" name="textLabel1_3" >
              <property name="text" >
               <string>Start value</string>
              </property>
              <property name="wordWrap" >
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="spacer3_2" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType" >
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>81</width>
                <height>21</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QSpinBox" name="startBox" >
              <property name="enabled" >
               <bool>false</bool>
              </property>
              <property name="minimum" >
               <number>0</number>
              </property>
              <property name="maximum" >
               <number>1023</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="Q3GroupBox" name="groupBox2" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title" >
        <string>Original (input) data</string>
       </property>
       <layout class="QVBoxLayout" >
        <item>
         <layout class="QVBoxLayout" >
          <item>
           <widget class="QCheckBox" name="delete1D" >
            <property name="text" >
             <string>delete input data after combination</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Q3Table" name="dataTable" >
            <property name="minimumSize" >
             <size>
              <width>0</width>
              <height>150</height>
             </size>
            </property>
            <property name="numRows" >
             <number>0</number>
            </property>
            <property name="numCols" >
             <number>3</number>
            </property>
            <column>
             <property name="text" >
              <string>1</string>
             </property>
            </column>
            <column>
             <property name="text" >
              <string>2</string>
             </property>
            </column>
            <column>
             <property name="text" >
              <string>3</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="errorLabel" >
       <property name="font" >
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text" >
        <string>...</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QPushButton" name="okButton" >
         <property name="text" >
          <string>OK</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="spacer5" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>141</width>
           <height>31</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="cancelButton" >
         <property name="text" >
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends>QGroupBox</extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3Table</class>
   <extends>Q3Frame</extends>
   <header>q3table.h</header>
  </customwidget>
  <customwidget>
   <class>Q3TextEdit</class>
   <extends>Q3Frame</extends>
   <header>q3textedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
